k8s-e2e-external-storage.groovy

```diff
@@ -153,13 +153,13 @@ node('cico-workspace') {
 				podman_pull(ci_registry, "docker.io", rook_ceph_cluster_image - d_io_regex)
 			}
 
-			// busy box is used in external storage testing
-			podman_pull(ci_registry, "docker.io", "library/busybox:1.29")
-			ssh "./podman2minikube.sh docker.io/library/busybox:1.29"
-
 			timeout(time: 30, unit: 'MINUTES') {
 				ssh "./single-node-k8s.sh --k8s-version=${k8s_release}"
 			}
+
+			// busy box is used in external storage testing
```

Comment:
@Rakshith-R I would suggest to remove the space between busy and box here, its a single image name

Reply:
This block has only moved it's position, and the text was not modified. So, I do not really care too much about this typo.

Reply:
Thanks for addressing this!